spec/fixtures/too_many_requests_error.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<IntuitResponse time="2016-12-07T15:27:09.527-08:00" xmlns="http://schema.intuit.com/finance/v3">
+  <Fault type="SERVICE">
+    <Error code="3001">
+      <Message>message=ThrottleExceeded; errorCode=003001; statusCode=429</Message>
```

Comment:
So odd to me that we introduced a new exception when they use the same message internally. What do you think about just having this rase a ThrottleExceeded instead of adding another thing to rescue?